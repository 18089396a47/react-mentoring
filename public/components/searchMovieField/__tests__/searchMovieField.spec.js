import React from 'react';
import renderer from 'react-test-renderer';
import SearchMovieField from '../searchMovieField';

jest.mock('../../searchButton/searchButton', () => 'SearchButton');
jest.mock('../../searchTypeToggle/searchTypeToggle', () => 'SearchTypeToggle');
jest.mock('../../searchInput/searchInput', () => 'SearchInput');

describe('SearchMovieField component', () => {
    it('SearchMovieField should', () => {
        const tree = renderer.create(
            <SearchMovieField />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
