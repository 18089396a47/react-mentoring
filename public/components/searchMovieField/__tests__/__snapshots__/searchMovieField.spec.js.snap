// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchMovieField component SearchMovieField should 1`] = `
<div
  className="search-movie-field"
>
  <SearchInput />
  <div
    className="search-movie-field__button-container"
  >
    <SearchTypeToggle />
    <SearchButton />
  </div>
</div>
`;
